language_en-US.js

```diff
@@ -56,8 +56,8 @@ module.exports = {
       REMOTE_CONTROL_DISALLOW: "The context cannot be remote-controlled.",
       DEVICE_NOT_CONTROLLABLE: "It is not possible to remote control the device.",
       VOLUME_CONTROL_DISALLOW: "It is not possible to remote control the device’s volume.",
```

Comment:
Speaking of contractions, how about this one?
```suggestion
      DEVICE_NOT_CONTROLLABLE: "It's not possible to remote control the device.",
      VOLUME_CONTROL_DISALLOW: "It's not possible to remote control the device’s volume.",
```

Reply:
Overall, it's looking good! Will merge once this nitpick is sorted out.

Reply:
Both "It's not" and "It isn't" work in this scenario, but I think emphasizing the word "not" is good. So sure, "It's not" sounds good.

Reply:
https://github.com/NicoWeio/Alexa-Spotify-Remote/pull/4/commits/200da3d302905a7aefdb4147bd94799249dfd344